src/services/CrawlPageProvince.ts

```diff
@@ -295,6 +295,16 @@ async function crawlDetailWarehouses() {
       const dateTimeRequest = (Date.now() - timeStartCrawl) / 1000;
       winston.info(`Crawl url ${dataUrl.url} done!. Time request: ${dateTimeRequest}s`);
     }
+
+    const crawlInfo = await getCrawlInfo();
+    const dataCrawlInfo = {
+      total: crawlInfo.totalUrl,
+      crawled: crawlInfo.crawledUrl,
+      remain: crawlInfo.remainUrl,
+      progress: crawlInfo.progress + '%',
+    };
+    await writeFile(`${FOLDER_FILE_DATA}/${STATISTICAL}`, JSON.stringify(dataCrawlInfo));
```

Comment:
Chỗ này có cần tạo thêm biến dataCrawlInfo không. Hay chỉ cần gán lại crawlInfo.progress, hoặc sửa luôn hàm getCrawlInfo()

Reply:
đã fix lấy luôn biến dataCrawlInfo

Reply:
Expected message:
 "message": "Crawling is in progress. Please wait until it is completed",
    "start_time": "YYYY/MM/DD, hh:mm:ss",
    "total": xxx,
    "crawled": yyy,
    "remain": zzz,
    "progress": "ttt%
    Khác với hàm getCrawlInfo